web_locators/change_password_page.py

```diff
@@ -0,0 +1,11 @@
+page_title="css:div.page-title"
+old_pwd_txt="css:input#OldPassword"
+new_pwd_txt="css:input#NewPassword"
+confirm_new_pwd_txt="css:input#ConfirmNewPassword"
+change_pwd_btn="css:button.change-password-button"
+bar_notification="xpath://div[@id='bar-notification']/div/p"
+bar_notification_x_btn="css:span.close"
+change_pwd_summary_errors="css:div.validation-summary-errors"
+old_pwd_error="css:span#OldPassword-error"
+new_pwd_error="css:span#NewPassword-error"
+confirm_new_pwd_error="css:span#ConfirmNewPassword-error"
```

Comment:
name is incorrect
It should be change_password_locators.py

Reply:
name has been changed.